File: OrchidCore/src/main/java/com/eden/orchid/impl/themes/tags/ScriptsTag.java
package com.eden.orchid.impl.themes.tags;

import com.eden.orchid.api.compilers.TemplateTag;
import com.eden.orchid.api.options.annotations.Description;

import javax.inject.Inject;
import javax.inject.Singleton;

@Singleton
@Description(value = "Render all script tags to the page.", name = "Scripts")
public final class ScriptsTag extends TemplateTag {

    @Inject
    public ScriptsTag() {
        super("scripts", Type.Simple, true);
    }

    @Override
    public String[] parameters() {
        return new String[]{};
    }

}

File: OrchidCore/src/main/kotlin/com/eden/orchid/impl/themes/tags/ScriptsTag.kt
package com.eden.orchid.impl.themes.tags

import com.eden.orchid.api.compilers.TemplateTag
import com.eden.orchid.api.options.annotations.Description

@Description(value = "Render all script tags to the page.", name = "Scripts")
class ScriptsTag : TemplateTag("scripts", Type.Simple, true) {

    override fun parameters(): Array<String> {
        return arrayOf()
    }

}
